tests/test_workdir.py

```diff
@@ -1,136 +1,46 @@
-        "/runs/fluxsite/analysis/bitwise-comparisons\n"
-        f"Creating task directories...\n"
-    )
+    # Success case: generate the full fluxsite directory structure
+    setup_fluxsite_directory_tree()
 
-    shutil.rmtree(MOCK_CWD / "runs")
+    for path in setup_mock_fluxsite_directory_list():
+        assert path.exists()
 
-    # Success case: test verbose output
-    with contextlib.redirect_stdout(io.StringIO()) as buf:
-        setup_fluxsite_directory_tree(
-            fluxsite_tasks=tasks, verbose=True, root_dir=MOCK_CWD
-        )
-    assert buf.getvalue() == (
-        f"Creating runs/fluxsite/logs directory: {MOCK_CWD}/runs/fluxsite/logs\n"
-        f"Creating runs/fluxsite/outputs directory: {MOCK_CWD}/runs/fluxsite/outputs\n"
-        f"Creating runs/fluxsite/tasks directory: {MOCK_CWD}/runs/fluxsite/tasks\n"
-        f"Creating runs/fluxsite/analysis directory: {MOCK_CWD}/runs/fluxsite/analysis\n"
-        f"Creating runs/fluxsite/analysis/bitwise-comparisons directory: {MOCK_CWD}"
-        "/runs/fluxsite/analysis/bitwise-comparisons\n"
-        f"Creating task directories...\n"
-        f"Creating runs/fluxsite/tasks/site_foo_R0_S0: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_foo_R0_S0\n"
-        f"Creating runs/fluxsite/tasks/site_foo_R0_S1: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_foo_R0_S1\n"
-        f"Creating runs/fluxsite/tasks/site_bar_R0_S0: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_bar_R0_S0\n"
-        f"Creating runs/fluxsite/tasks/site_bar_R0_S1: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_bar_R0_S1\n"
-        f"Creating runs/fluxsite/tasks/site_foo_R1_S0: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_foo_R1_S0\n"
-        f"Creating runs/fluxsite/tasks/site_foo_R1_S1: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_foo_R1_S1\n"
-        f"Creating runs/fluxsite/tasks/site_bar_R1_S0: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_bar_R1_S0\n"
-        f"Creating runs/fluxsite/tasks/site_bar_R1_S1: "
-        f"{MOCK_CWD}/runs/fluxsite/tasks/site_bar_R1_S1\n"
-    )
+    shutil.rmtree(Path("runs"))
 
-    shutil.rmtree(MOCK_CWD / "runs")
 
-
-def test_clean_directory_tree():
+@pytest.mark.parametrize("test_path", [Path("runs"), Path("src")])
+def test_clean_directory_tree(test_path):
     """Tests for `clean_directory_tree()`."""
     # Success case: directory tree does not exist after clean
-    tasks = setup_mock_tasks()
-    setup_fluxsite_directory_tree(fluxsite_tasks=tasks, root_dir=MOCK_CWD)
-
-    clean_directory_tree(root_dir=MOCK_CWD)
-    assert not Path(MOCK_CWD, "runs").exists()
-
-    setup_src_dir(root_dir=MOCK_CWD)
-    clean_directory_tree(root_dir=MOCK_CWD)
-    assert not Path(MOCK_CWD, "src").exists()
-
-
-def test_setup_src_dir():
-    """Tests for `setup_src_dir()`."""
-    # Success case: make src directory
-    setup_src_dir(root_dir=MOCK_CWD)
-    assert Path(MOCK_CWD, "src").exists()
+    mkdir(test_path)
```

Comment:
Small nit-pick but I think its better if we use the `Path.mkdir()`. Then this test won't fail in case `mkdir()` in fs.py breaks.
```suggestion
    test_path.mkdir()
```